02-spatial-data.Rmd

```diff
@@ -145,6 +145,14 @@ For more advanced map making we recommend using a dedicated visualisation packag
 - map export 
 -->
 
+
+<!--Maybe show also somewhere that `world[0]` produces only a plot of the geometry which is rather useful if you do not want to plot a specific attribute. This way, you can for example dismiss the col = "white"-argument in your Nigeria example.
```

Comment:
`plot(world$geom)` is even more intuitive than `plot(world[0])`, and exactly doing the same, i.e., just plotting the geometry,